Makefile

```
@@ -25,6 +24,7 @@ YQ ?= $(LOCALBIN)/yq
 
 KUSTOMIZE_INSTALL_SCRIPT ?= "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"
 YQ_INSTALL_SCRIPT ?= https://github.com/mikefarah/yq/releases/latest/download/yq_$(OS)_$(ARCH)
+HELM_DOCS_REPO ?= github.com/norwoodj/helm-docs/cmd/helm-docs@latest
```

I'd like to pin the version and not use `latest` because it could break things when there is a change
```suggestion
HELM_DOCS_REPO ?= github.com/norwoodj/helm-docs/cmd/helm-docs
HELM_DOCS_VERSION ?= 1.14.2
```